README.md

```diff
@@ -31,6 +66,22 @@ provider "kentik-synthetics" {
 }
 ```
 
+or environment variables:
+
+```bash
+export KTAPI_AUTH_EMAIL="john@acme.com"
+export KTAPI_AUTH_TOKEN="token123"
+export KTAPI_URL="http://localhost:8080" # custom apiserver
+```
+
+2. Invoke:
+
+```bash
+terraform init
+terraform apply
+```
+
+
 ## Development
 
```

Comment:
I think it's time to remove `### Development state` chapter, since acceptance tests will probably not be implemented in the near future and rest is done

Reply:
Agreed. I actually prefer to not have this section at all.

Reply:
Addressed: 2f731aa50d3f62d52c77554608a7a730bb8f6007